script.lazytv/resources/addon_clone.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
can you explain why this functionality exists?

Reply:
This was actually a question raised when LazyTV was first added to the repo, and was remedied by having the user explicitly permit the local system changes.
LazyTV is a service which determines the next episode to watch in each TV Show; it keeps up to date with your viewing and changes the episodes accordingly. LazyTV is also a script which will either show those episodes as a list or start playing a randomly selected collection from them. Click a button, the script does its thing. LazyTV allows the user to restrict the show list to specific TV Shows of their choosing. Sometimes the user wants two different show lists, such as a Kids list and a Grown-Ups list. The cloning functionality allows the user to copy the LazyTV script and assign a different list to it. Because these are separate addons, they can very easily put them into different widget positions on the front page. These local changes are made only when called by and after explicit permission is granted by the user.

Reply:
Yeap, sorry for bringing this again. We can't remember every single addon/discussion :)
It's fine to keep it as long as is opt-in.

Reply:
Not an issue at all, thanks for the detailed code review.